src/lib/ibcclient.spec.ts

```diff
@@ -251,6 +251,15 @@ test.serial('tests parsing with multi-message', async (t) => {
   const { logs: sendLogs } = await nodeA.sendTokens(srcAddr, [
     { amount: '5000', denom: simapp.denomFee },
   ]);
+  t.assert(
+    logger.verbose.calledWithMatch(/Send tokens/),
+    logger.verbose.callCount.toString()
+  );
+  t.assert(
+    logger.debug.calledWithMatch(/Send tokens/),
```

Comment:
Shouldn't these be somewhat different strings to test well?

verbose: "Send tokens to"
debug: "Send tokens:"